src/node_file-inl.h

```diff
@@ -123,6 +123,41 @@ v8::Local<v8::Value> FillGlobalStatsArray(Environment* env,
   }
 }
 
+template <typename NativeT, typename V8T>
+void FillStatfsArray(AliasedBufferBase<NativeT, V8T>* fields,
+                     const uv_statfs_t* s) {
+#define SET_FIELD(field, stat)                                                \
+  fields->SetValue(static_cast<size_t>(FsStatfsOffset::field),                \
+                   static_cast<NativeT>(stat))
+
+  SET_FIELD(kType, s->f_type);
+  SET_FIELD(kBSize, s->f_bsize);
+  SET_FIELD(kBlocks, s->f_blocks);
+  SET_FIELD(kBFree, s->f_bfree);
+  SET_FIELD(kBAvail, s->f_bavail);
+  SET_FIELD(kFiles, s->f_files);
+  SET_FIELD(kFFree, s->f_ffree);
+  SET_FIELD(kSpare1, s->f_spare[0]);
+  SET_FIELD(kSpare2, s->f_spare[1]);
+  SET_FIELD(kSpare3, s->f_spare[2]);
+  SET_FIELD(kSpare4, s->f_spare[3]);
+#undef SET_FIELD
+}
+
+v8::Local<v8::Value> FillGlobalStatfsArray(Environment* env,
+                                          const bool use_bigint,
+                                          const uv_statfs_t* s) {
```

Comment:
Minor style issue: can you line up the arguments? The indent is off by one space now.